.github/workflows/build-test-prValidation.yml

```diff
@@ -43,10 +41,10 @@ jobs:
           -Dsbt.override.build.repos=false \
           -Dsbt.log.noformat=false \
           -Dakka.log.timestamps=true \
-          scalafmtCheckAll scalafmtSbtCheck
+          scalafmtCheckAll scalafmtSbtCheck headerCheckAll
```

Comment:
This was running on Travis CI before, together with MiMa reports. I think it is better to have it as part of the code style checks, though.